pkg/k8sutils/templates/pod/spec.go

```diff
@@ -423,3 +406,38 @@ func Annotations(spec v1.SpecInterface, clusterAnnotations map[string]string, no
 	}
 	return annotations
 }
+
+func PodSecurityContext(spec v1.SpecInterface) *corev1.PodSecurityContext {
+	userId := int64(1000)
+	groupId := int64(1000)
+	onRootMismatch := corev1.FSGroupChangeOnRootMismatch
```

Comment:
what's the default value of this `onRootMismatch`? Seems this config is not exposed, I suppose the end user can't control this behavior.

Reply:
Always is the default value 
```
	// FSGroupChangeAlways indicates that volume's ownership and permissions
	// should always be changed whenever volume is mounted inside a Pod. This the default
	// behavior.
	FSGroupChangeAlways PodFSGroupChangePolicy = "Always"

	// FSGroupChangeOnRootMismatch indicates that volume's ownership and permissions will be changed
	// only when permission and ownership of root directory does not match with expected
	// permissions on the volume. This can help shorten the time it takes to change
	// ownership and permissions of a volume.
	FSGroupChangeOnRootMismatch PodFSGroupChangePolicy = "OnRootMismatch"
```

Reply:
ok